doc/ingest.md

```diff
@@ -86,6 +88,29 @@ The network indexer nodes expect that metadata begins with a `uvarint` identifyi
   * the proposed `uvarint` protocol is `0x3D0000`.
   * the following bytes are not yet defined.
 
+#### ExtendedMetadata
+
+The `ExtendedMetadata` field allows for specification of provider families, in cases where a provider operates multiple PeerIDs, perhaps with different access methods between them, but over the same database of content.
+
+```
+type ExtendedMetadata struct {
+    Family [ExtendedProvider]
+}
+
+type ExtendedProvider struct {
+    Provider String
+    Addresses [String]
+    Metadata optional Bytes
+    Signature Bytes
+}
+```
+
+* If Metadata is not specified for an `ExtendedProvider`, the metadata for an individual `Advertisement` will be used instead.
+* If an `ExtendedProvider` listing is written with no `ContextID`, those peers will be returned forall published advertisements for the advertisement Publisher.
+* If an `ExtendedProvider` listing is written on an advertisement with a `ContextID`, it will over-ride any chain-globals, and that specific set of records will be returned for that context ID.
+* The `Signature` for an `ExtendedProvider` is made using the key of that `Provider` over the serialized `Advertisement` containing the following subset of fields:
+    * `PreviousID`, `Provider`, `Addresses`, `Entries`, `ContextID`, `Metadata`.
```

Comment:
Do we need `Addresses` and `Entries` to be signed over given that those are ignored if `ExtendedMetadata` is present?

Reply:
updated signature to be safer